packages/@react-spectrum/color/test/ColorArea.test.tsx

```diff
@@ -458,12 +458,13 @@ describe('ColorArea', () => {
     });
 
     it('the slider is focusable', () => {
```

Comment:
With the number of new stories being assed for HSB and HSL I expected to see more tests.

Reply:
The stories are mostly to check the gradients, which is also why they're all in chromatic, but since the gradients can change in usage, that's why they're also here
there isn't a lot of actual new functionality being added here, at some point I'd love to split out the ColorArea from the other components and make like a ColorPicker section that joins them all together
Otherwise tests are mostly covered over in ColorWheel/ColorSlider/ColorField
I'll add one for HSB and HSL though